packages/node_modules/nav-frontend-hjelpetekst/src/hjelpetekst.tsx

```diff
@@ -214,6 +218,9 @@ class HjelpetekstBase extends React.Component<HjelpetekstProps, State> {
         const Hjelpetekst = this.state.erSynlig ? this.renderContent() : null;
         const Anchor = this.props.anchor;
         const ariaId = this.getAriaId();
+        const tekstBeskrivelse = this.props.tekstBeskrivelse;
+        const TekstBeskrivelse = tekstBeskrivelse ? <Normaltekst>{tekstBeskrivelse}</Normaltekst> : null;
```

Comment:
Lurer på om "beskrivelse" er litt upresis begrepsbruk her - som lett kan forveksles med `children` ettersom komponentens funksjon i utgangspunktet er å gi en beskrivelse/hjelpetekst på noe annet. Tror jeg hadde foretrukket "label" her i stedet.

Samtidig er det litt uheldig å ha så like variabel-navn som `tekstBeskrivelse` og `TekstBeskrivelse` på to forskjellige ting. Foreslår at du endrer den nye prop'en til å hete `labelTekst` i stedet. Lurer også på om du kan droppe flere av disse variablene (inkl. `Hjelpetekst`) ved å heller lage en `renderLabel`-metode på samme måte som `renderContent`. Da kan du forenkle både linje 249 til `{this.props.labelTekst && this.renderLabel()}` og 257 til `{this.state.erSynlig && this.renderContent()}` - som jeg synes blir mye enklere å lese.

Reply:
👍 Jeg har commitet og pushet endringene du foreslo 😄